meta-mender-core/recipes-mender/mender-client/mender-client.inc

```diff
@@ -66,49 +61,43 @@ FILES:mender-modules-gen = " \
 SYSROOT_DIRS += "/data"
 
 SRC_URI:append:mender-image:mender-systemd = " \
-    file://mender-client-data-dir.service \
+    file://mender-data-dir.service \
 "
 
 SRC_URI:append:mender-persist-systemd-machine-id = " \
-    file://mender-client-systemd-machine-id.service \
-    file://mender-client-set-systemd-machine-id.sh \
+    file://mender-systemd-machine-id.service \
+    file://mender-set-systemd-machine-id.sh \
 "
 
 SRC_URI:append:mender-growfs-data:mender-systemd = " \
-    file://mender-client-resize-data-part.sh.in \
+    file://mender-resize-data-part.sh.in \
     file://mender-grow-data.service \
     file://mender-systemd-growfs-data.service \
 "
 
 FILES:${PN}:append:mender-image:mender-systemd = " \
-    ${systemd_unitdir}/system/${MENDER_CLIENT}-data-dir.service \
-    ${systemd_unitdir}/system/${MENDER_CLIENT}.service.wants/${MENDER_CLIENT}-data-dir.service \
+    ${systemd_unitdir}/system/mender-data-dir.service \
+    ${systemd_unitdir}/system/${MENDER_CLIENT}.service.wants/mender-data-dir.service \
 "
 
 FILES:${PN}:append:mender-growfs-data:mender-systemd = " \
-    ${bindir}/mender-client-resize-data-part \
+    ${bindir}/mender-resize-data-part \
     ${systemd_unitdir}/system/mender-grow-data.service \
     ${systemd_unitdir}/system/mender-systemd-growfs-data.service \
     ${systemd_unitdir}/system/data.mount.wants/mender-grow-data.service \
     ${systemd_unitdir}/system/data.mount.wants/mender-systemd-growfs-data.service \
 "
 
 FILES:${PN}:append:mender-persist-systemd-machine-id = " \
-    ${systemd_unitdir}/system/mender-client-systemd-machine-id.service \
-    ${systemd_unitdir}/system/${MENDER_CLIENT}.service.wants/mender-client-systemd-machine-id.service \
-    ${bindir}/mender-client-set-systemd-machine-id.sh \
+    ${systemd_unitdir}/system/mender-systemd-machine-id.service \
+    ${systemd_unitdir}/system/${MENDER_CLIENT}.service.wants/mender-systemd-machine-id.service \
+    ${bindir}/mender-set-systemd-machine-id.sh \
 "
 
-# Go binaries produce unexpected effects that the Yocto QA mechanism doesn't
-# like. We disable those checks here.
-INSANE_SKIP:${PN} = "ldflags textrel"
-INSANE_SKIP:${PN}-ptest = "ldflags textrel"
-
-GO_IMPORT = "github.com/mendersoftware/mender"
-
 _MENDER_PACKAGECONFIG_DEFAULT = "${@mender_feature_is_enabled("mender-client-install", "dbus modules inventory-network-scripts", "", d)}"
 PACKAGECONFIG ?= "${_MENDER_PACKAGECONFIG_DEFAULT}"
 
+# TODO - How should this be split (?)
```

Comment:
They are all related to `mender-updated`, right? `mender-authd` has nothing to do with "client install" or bootloaders.